<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyEditMesType</class>
 <widget class="QDialog" name="lyEditMesType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lyEditMesType</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DataManager.qrc">
    <normaloff>:/DataManger/Data/image/calendar.png</normaloff>:/DataManger/Data/image/calendar.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="mesTypeLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>会议类型</string>
      </property>
     </widget>
     <widget class="QComboBox" name="mesTypeComboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>51</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="modifyLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>需要修改</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifyEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加新类型</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除当前类型</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>修改当前类型</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../DataManager.qrc"/>
 </resources>
 <connections/>
</ui>
